pkg/parsers/kernel/kernel.go

```diff
@@ -43,7 +44,8 @@ func CompareKernelVersion(a, b *KernelVersionInfo) int {
 	return 0
 }
 
-func GetKernelVersion() (*KernelVersionInfo, error) {
+// GetKernelVersion get the current kernel version
+func GetKernelVersion() (*VersionInfo, error) {
```

Comment:
Few lines later we return a nil (``return nil, err``), I think that's why the pointer was there.